src/Domain/Donation.php

```diff
@@ -15,6 +15,8 @@
 use Messages;
```

Comment:
We just read the section on "Partially Immutable Entities" in book club. I took it as good advice to start by making all properties of everything immutable and then only allow mutations for things that we actually want to change. We never want to change the amount or the currency on any donation in our system.

This is the only file used in in production changed in this PR.